src/impls.rs

```diff
@@ -264,8 +264,10 @@ safety_comment! {
     ///   clear how we'd prove it short of adding text to the stdlib docs that
     ///   says so explicitly, which likely wouldn't be accepted.
     ///
-    /// [1] https://doc.rust-lang.org/stable/std/num/struct.NonZeroU8.html
-    /// [2] https://doc.rust-lang.org/stable/std/num/struct.NonZeroI8.html
+    /// [1] https://doc.rust-lang.org/1.81.0/std/num/type.NonZeroU8.html
+    ///     `NonZeroU8 is guaranteed to have the same layout and bit validity as u8 with
+    ///     the exception that 0 is not a valid instance`
+    /// [2] https://doc.rust-lang.org/1.81.0/std/num/type.NonZeroI8.html
```

Comment:
```suggestion
    /// [1] Per https://doc.rust-lang.org/1.81.0/std/num/type.NonZeroU8.html:
    ///
    ///     `NonZeroU8` is guaranteed to have the same layout and bit validity as `u8` with
    ///     the exception that 0 is not a valid instance
    ///
    /// [2] https://doc.rust-lang.org/1.81.0/std/num/type.NonZeroI8.html
```

The backticks are used around code items like `NonZeroU8` or `u8`, not around the entire quote. Also, add a newline before (and in the case where there's another citation afterward, a newline after as well).